Sources/Plasma/NucleusLib/pnUUID/pnUUID_Unix.cpp

```diff
@@ -133,13 +120,13 @@ bool plUUID::FromString( const char * str )
     return true;
 }
 
-bool plUUID::ToString( ST::string & out ) const
+bool plUUID::ToString(ST::string& out) const
 {
     uuid_t g;
-    plUUIDHelper::CopyToNative( g, this );
+    plUUIDHelper::CopyToNative(g, this);
     char buf[40];
-    uuid_unparse( g, buf );
-    out = buf;
+    uuid_unparse(g, buf);
+    out = ST::string(buf).to_lower();
```

Comment:
There seems to be an `uuid_unparse_lower` function in `<uuid/uuid.h>` on both [Linux](https://github.com/util-linux/util-linux/blob/e3192bfd1dd129c70f5416e1464135d8cd22c956/libuuid/src/uuid.h#L108) and [macOS](https://github.com/apple-oss-distributions/xnu/blob/94d3b452840153a99b38a3a9659680b2a006908e/bsd/uuid/uuid.h#L74), which is defined to return lowercase specifically. Can we use that directly?

Reply:
Yep! Good suggestion